lib/new.js

```diff
@@ -43,6 +43,20 @@ function getPackageName(name) {
   return `skyux-${prefix}-${name}`;
 }
 
+/**
+ * Returns a string to be used for the description property in package.json/logger.
+ * First letter is capitalized - so expected to be used first in a sentence.
+ * @name getPackageDescriptionType
+ */
+function getPackageDescriptionType(settings) {
+  switch (settings.template.name) {
+    case 'library':
+      return 'library';
+    default:
+      return 'single-page application (SPA)';
```

Comment:
I missed this yesterday, but do we really need "(SPA)" here? I don't think so.

Reply:
I think so.  If anything, I'd rather call it a SPA more frequently than a "single-page application."